remo/remozilla/tasks.py

```diff
@@ -24,12 +24,13 @@
                    u'status', u'assigned_to', u'resolution',
                    u'last_change_time', u'flags', u'comments']
 
-URL = ('https://api-dev.bugzilla.mozilla.org/latest/bug'
-       '?username={username}&password={password}&'
-       'product=Mozilla%20Reps&component={component}&'
-       'include_fields={fields}&changed_after={timedelta}d&'
-       'offset={offset}&limit={limit}')
-
+URL = 'https://bugzilla.mozilla.org/rest/bug'
+LOGIN_URL = ('https://bugzilla.mozilla.org/rest/login?login={username}'
+             '&password={password}')
+URL_PARAMS = ('&product=Mozilla%20Reps&component={component}&'
+              'include_fields={fields}&changed_after={timedelta}d&'
+              'offset={offset}&limit={limit}')
+TOKEN_PARAM = '?token={token}'
```

Comment:
I'd merge this with URL_PARAMS because you're only using this together and you end up urlmerging this.